Test/logger/TextLogger.dfy

```diff
@@ -0,0 +1,20 @@
+// RUN: %dafny /compile:0 /verificationLogger:text /vcsSplitOnEveryAssert "%s" > "%t"
```

Comment:
Great test, and I continue to appreciate having support for the OutputCheck DSL. :)

We should also have a test that DOESN'T use `/vcsSplitOnEveryAssert` too, so we know we handle the case of more than one assertion in a batch.

Reply:
Oh, yes!